heltour/tournament/templates/tournament/availability.html

```diff
@@ -45,13 +45,13 @@ <h3>Edit Availability</h3>
                                 <tr>
                                     <td>Round {{ r.number }}</td>
                                     <td>{{ r.start_date|date_or_q:'%b %-d' }} - {{ r.end_date|date_or_q:'%b %-d' }}</td>
-                                    {% for p, is_av, is_red in av_list %}
+                                    {% for p, is_av, is_fixed in av_list %}
```

Comment:
I don't remember what this variable was supposed to indicate in the previous version, but the new name also doesn't seem particularly clear to me. 

Is it necessary for this branch?

Reply:
That parameter controls if the availability checkbox can be changed or not. I have renamed it to "cannot_change" in the upcoming commit. I know that using negations in boolean variables is usually not very elegant but here I think it conveys the usage.

Reply:
What about `is_readonly` or `is_immutable`.  I don't mind `cannot_change`, but these are other potential names. I'm happy to merge it as is, though.

Reply:
I like is_immutable bebest and will adapt it.